apps/ligo-virgo/src/pages/workspace/docs/components/toc/toc.css

```diff
@@ -0,0 +1,13 @@
+.toc-scroll-item {
+    animation: blinker 1s linear;
+    animation-delay: 1s;
+}
+
+@keyframes blinker {
+    0% {
+        background: rgba(152, 172, 189, 0.1);
+    }
+    100% {
+        background: rgba(152, 172, 189, 0.1);
+    }
+}
```

Comment:
You can use `styled` like this

```tsx
const Blinker = styled('div')``;

// ---
const className = Blinker.toString();

block.dom.classList.add(className);

```

Reply:
I need more details example about keyframes styed usage